addons/pharma/functions/fnc_treatmentAdvanced_PervitinLocal.sqf

```diff
@@ -0,0 +1,395 @@
+ *
+ * Public: No
+ */
+
+params ["_patient"];
+
+if (ACE_Player != _patient) exitWith {};
+
+/// ACE Fatigue 
+if (ACEGVAR(advanced_fatigue,enabled)) then {
+	
+	[{
+		params ["_patient"];
+
+		if !(alive _patient) exitWith {};
+		ace_advanced_fatigue_anReserve = ace_advanced_fatigue_anReserve + 3000;
+		["PDF", 0] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_start), 2, _patient] call ACEFUNC(common,displayTextStructured); 
+	},
+	[_patient], 10] call CBA_fnc_waitAndExecute;
+
+
+	[{
+		params ["_patient"];
+
+		if !(alive _patient) exitWith {};
+		["PDF"] call ace_advanced_fatigue_fnc_removeDutyFactor;
+		[LLSTRING(Pervitin_mid), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient], 180] call CBA_fnc_waitAndExecute; /// 3m
+
+
+	[{
+		params ["_patient"];
+
+		if !(alive _patient) exitWith {};
+		ace_advanced_fatigue_anReserve = ace_advanced_fatigue_anReserve + 3000;
+		["PDF", 0.4] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_mid2), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient], 240] call CBA_fnc_waitAndExecute; /// 4m
+
+
+	[{
+		params ["_patient"];
+
+		if !(alive _patient) exitWith {};
+		ace_advanced_fatigue_anReserve = ace_advanced_fatigue_anReserve + 3000;
+		["PDF", 0.6] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_mid3), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient], 360] call CBA_fnc_waitAndExecute; /// 6m
+
+
+	[{
+		params ["_patient"];
+
+		if !(alive _patient) exitWith {};
+		["PDF", 2] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_mid4), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient], 510] call CBA_fnc_waitAndExecute; /// 8:30m
+
+
+	[{
+		params ["_patient"];
+
+		if !(alive _patient) exitWith {};
+		["PDF"] call ace_advanced_fatigue_fnc_removeDutyFactor;
+		[LLSTRING(Pervitin_end), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient], 600] call CBA_fnc_waitAndExecute; /// 10m
+
+
+	/// ACE Fatigue Weapon Sway
+
+	if (GVAR(weapon_sway_pervitin)) then {
+
+		if (isNil GVAR(originalSwayFactor)) then {
+			GVAR(originalSwayFactor) = ace_advanced_fatigue_swayFactor;
+		};
+
+		[{
+			params ["_patient"];
+			if !(alive _patient) exitWith {};
+			ace_advanced_fatigue_swayFactor = GVAR(originalSwayFactor) * 0.3;
+		},
+		[_patient], 15] call CBA_fnc_waitAndExecute;
+
+
+		[{
+			params ["_patient"];
+			if !(alive _patient) exitWith {};
+			ace_advanced_fatigue_swayFactor = GVAR(originalSwayFactor) * 0.5 ;
+		},
+		[_patient], 60] call CBA_fnc_waitAndExecute;
+
+		
+		[{
+			params ["_patient"];
+			if !(alive _patient) exitWith {};
+			ace_advanced_fatigue_swayFactor = GVAR(originalSwayFactor) * 1;
+		},
+		[_patient], 90] call CBA_fnc_waitAndExecute;
+
+
+		[{
+			params ["_patient"];
+			if !(alive _patient) exitWith {};
+			ace_advanced_fatigue_swayFactor = (GVAR(originalSwayFactor) * 1.3) min 2;
```

Comment:
Im too lazy to mention all.  
Replace "ace_advanced_fatigue_swayFactor" with "ACEGVAR(advanced_fatigue, swayFactor)"